husky_services/expression-language.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+define(['underscore', 'services/husky/logger'], function(_, Logger) {
+
+    'use strict';
+
+    var instance,
+        getInstance = function() {
+            if (instance == null) {
+                instance = new ExpressionLanguage();
+            }
+
+            return instance;
+        };
+
+    function ExpressionLanguage() {
```

Comment:
Is `ExpressionLanguage` really a matching term? We are only evaluating based on javascript object and not introducing another kind of language.